apps/condo/lang/es/es.json

```diff
@@ -7,7 +7,7 @@
   "Add": "Añadir",
   "AddContact": "Añadir propietario",
   "AddEmployee": "Añadir empleado",
-  "AddHoldingEmployee": "Añadir empleado a holding",
+  "AddHoldingEmployee": "Añadir empleado a organización",
```

Comment:
Hey, it is great to see you PR 

Holdings are a separate entity from a technical point of view, are you sure about this change?